src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -137,6 +137,11 @@ public void sortListByGrade(boolean isInAscending) {
         this.addressBook.sortByGrade(isInAscending);
     }
 
+    @Override
+    public Predicate<Person> createPersonIsInClassPredicate(String studentClass) {
+        return person -> person.getStudentClass().isSameClass(studentClass);
+    }
+
```

Comment:
Can use this in the MarkAttendanceCommand right?

Reply:
Ya I tried but then it didn't work leh that's why I commented out some code using this in the MarkAttendanceCommand class